ray-operator/apis/ray/v1/rayjob_types.go

```diff
@@ -87,15 +87,84 @@ const (
 
 type DeletionPolicyType string
 
```

Comment:
Can you help me move the DeletionPolicyType strings to here? thank you!
```suggestion
type DeletionPolicyType string

const (
	DeleteCluster DeletionPolicyType = "DeleteCluster" // To delete the entire RayCluster custom resource on job completion.
	DeleteWorkers DeletionPolicyType = "DeleteWorkers" // To delete only the workers on job completion.
	DeleteSelf    DeletionPolicyType = "DeleteSelf"    // To delete the RayJob custom resource (and all associated resources) on job completion.
	DeleteNone    DeletionPolicyType = "DeleteNone"    // To delete no resources on job completion.
)

```